x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -453,6 +455,31 @@ public void testInvalidPolicyNames() throws UnsupportedEncodingException {
         assertThat(ex.getMessage(), containsString("invalid policy name"));
     }
 
+    public void testDeletePolicyInUse() throws IOException {
+        String managedIndex1 = randomAlphaOfLengthBetween(5, 15).toLowerCase(Locale.ROOT);
+        String managedIndex2 = randomAlphaOfLengthBetween(5, 15).toLowerCase(Locale.ROOT);
+        String unmanagedIndex = randomAlphaOfLengthBetween(5, 15).toLowerCase(Locale.ROOT);
```

Comment:
I know the odds are unlikely, but a subset of these may end up being the same index name